### ISSUE TITLE
Add optional shortcuts for opening and closing parachute

### ISSUE THREAD

**Issue by BananaLoaf** [tcorreabr/Parachute#113]
I use touchegg for touchpad gestures and I have assigned three finger swipe up and three finger swipe down for toggling parachute. The problem is that both gestures do the same thing and swiping up/down to open and then swiping the same direction to close does not feel natural. Moreover, due to touchegg nature, swiping up half way would open parachute and swiping up the rest of the way would close it and I think that delegated shortcuts for opening/closing would solve that issue. I humbly ask you to implement said feature.

**Comment by koma111** [comment]
I think the best way to do this would be to have separate shortcuts indeed, but set the same combination for both by default, and tell the user in README, to set both to the same key combo to make it work as a toggle.

**Comment by KINGSABRI** [comment]
The toggle behavior makes sense (one press opens, same press closes)

**Comment by koma111** [comment]
> The toggle behavior makes sense (one press opens, same press closes)

Yes, on desktop, it does. But that can be replicated by having separate shortcuts for opening and closing, and by default setting them to THE SAME KEY COMBO. This way, users who want the default toggle behavior suspect nothing, but laptop/touchpad users, who want to use touchpad gestures, can bind them to the two different actions.
Having the cake and eating it too.

**Comment by KINGSABRI** [comment]
Oh, totally agreed. I didn't think about the mouse-pad gesture. it would be really cool to match macOS gesture
